Insurance_company/insurance_app/models.py

```diff
@@ -28,6 +36,7 @@ class CarInsurance(models.Model):
     )
 
     # predefined fields
+    policy_id = models.CharField(generate_id(), primary_key=True)
```

Comment:
czemu tak?

Reply:
Dziś sobie to przegadamy w grupie, ja pomyślałam aby w tym miejscu użyć policy_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False) bo chcemy by był on generowany automatycznie przy tworzeniu nowej polisy i  unikalny dla każdej. UUIDField generuje unikalne identyfikatory w sposób losowy, zapewniając tym samym unikalność dla każdej polisy i umożliwiając generowanie identyfikatorów podczas tworzenia nowych obiektów. Myślisz @adam-harmasz  że to byłoby ok?

Reply:
@adam-harmasz poprawione na models.UUIDField, zrezygnowałem z funkcji generate_id()